client/src/components/EventView/Description/index.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import styled from "styled-components";
+
+const TextBar = styled.div`
+font-size: 2.2vw;
+font-weight: bold;
+padding: 2% 0%;
+`;
+
+//takes in a description
+const Description = ({description,...props}) => {
+return <div>
+    <TextBar>Description</TextBar>
```

Comment:
Instead of this TextBar, you can use the <h1>. That will match what we have done in other places.